src/test/components/NetworkChart.test.js

```diff
@@ -77,6 +97,28 @@ describe('NetworkChart', function() {
 
     delete window.devicePixelRatio;
   });
+
+  it('merges NetworkMarkers correctly', () => {
+    window.devicePixelRatio = 1;
+
+    const profile = getProfileWithMarkers([...NETWORK_START_END_MARKERS]);
+    const {
+      flushRafCalls,
+      dispatch,
+      networkChart,
+      flushDrawLog,
+    } = setupWithProfile(profile);
+
+    dispatch(changeSelectedTab('network-chart'));
+    networkChart.update();
+    flushRafCalls();
+
+    const drawCalls = flushDrawLog();
+    expect(networkChart).toMatchSnapshot();
```

Comment:
I think this test would work much better as an Enzyme test rather than a snapshot test. This way we can assert specific properties of the component, rather than relying on overly broad snapshots. Originally we used `react-test-renderer` for all the tests, but Enzyme provides many nicer features.

I'm updating our docs to with more information on our latest usage of Enzyme. I'll link to them here after I open a PR with it.

Reply:
See #1414